src/ens.ts

```diff
@@ -29,6 +29,19 @@ export default class Ens {
     );
   }
 
+  /* Test functions bellow */
+
+  _resolverCallToName = (resolverContract, nodeHash) =>
```

Comment:
Use the same method definition format as in other places and separate methods with one blank line